.github/workflows/backend_unit_test.yml

```
@@ -0,0 +1,44 @@
+      docker:
+        image: docker:19.03.12
+        options: --privileged
+        ports:
+          - 5432:5432
+          - 8000:8000
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # GitHubリポジトリの設定で定義したシークレットを.envファイルに書き込む
+      - nmae: Create .env file
```

actionlint (1.7.4)

unexpected key "nmae" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)